### PR TITLE
Modify setup.yml to setup dependencies for the Sharp NPM package

### PR DESCRIPTION
**Pull request by bschwind**
I'm using a package called [Sharp](https://github.com/lovell/sharp) to resize uploaded user images. To run as fast as it does, it depends on the libvips image processing library. This pull request adds libvips to the vagrant developer environment.

@arowM, we probably need to talk about the best way to get this working on production environments as well. Should be pretty simple if the environments aren't too different.

### CODE REVIEW

**Comment by bschwind** [comment]
@arowM, please review

**Comment by bschwind** [comment]
@shunjikonishi, please review

File: setup.yml [hunk 1]
```diff
@@ -63,6 +63,18 @@
...
+    # Install pkg-config (needed to preinstall Sharp)
+    #####################################
+    - name: Install pkg-config
+      apt: name=pkg-config
+
+    #####################################
+    # Setup sharp (needed by attache-node-api for image resizing)
+    #####################################
+    - name: Preinstall Sharp
+      shell: curl -s https://raw.githubusercontent.com/lovell/sharp/master/preinstall.sh | bash -
```
Comment (shunjikonishi) [inline setup.yml]: Is it better to use master?
Though I don't know about this library much, it might be better to use tagged(released) version.

**Comment by shunjikonishi** [comment]
I commented in a note.
However, this is almost looks good.

LGTM

File: setup.yml (hunk 1 above)
Comment (bschwind) [inline setup.yml]: Oh that's a good point! I'll change it to be the tag of the version I'm using in package.json

**Pull request merged**